### DISCUSSION TITLE
Netlify 部署后 Iconfont 图标丢失

### DISCUSSION THREAD

**Discussion by Masterain98** [vuepress-theme-hope/vuepress-theme-hope#2987] [Q&A] [answered]
## 问题
在[这次PR](https://github.com/DGP-Studio/Snap.Hutao.Docs/pull/26)后，发现 Netlify 部署的结果中不包含任何 Iconfont 图标。但是在本地调，以及 GitHub Pages 的部署结果中都是正常的。

Netlify 部署结果： https://deploy-preview-26--snaphutao.netlify.app/en/features/
Netlify 部署日志：https://app.netlify.com/sites/snaphutao/deploys/6427ae60beabdd0007bacb04
GitHub Pages: https://pages.hut.ao

## 截图
![image](https://user-images.githubusercontent.com/10614984/229449876-6f4c8752-ec60-455d-9433-880ab93b22f4.png)
![image](https://user-images.githubusercontent.com/10614984/229449949-fa1387da-d1bc-46f1-a937-a9f36e100b04.png)

**Comment by Mister-Hope** [comment, marked as answer]
I am not sure, but I also met similar issues with iconfont, actually the cdn link it provides is for test only, and you should self-host the assets in production.

**Comment by Masterain98** [comment]
@Mister-Hope 
I got it fixed by myself. The problem is caused by Netlify's **Asset Optimization** feature. I tried to compare two deploy version between Netlify and GitHub Pages and found the `framework-caa722f8.js` file on Netlify was headed to [cloudfront](https://d33wubrfki0l68.cloudfront.net/js/1463358591e5adb5d22ed129c8ced4dbad794443/assets/framework-caa722f8.js), which has 404 error.

After disabling the Asset Optimization, the issue was gone.

![image](https://user-images.githubusercontent.com/10614984/229649666-4932d6db-4774-4d20-934e-a5e0546981e0.png)
![image](https://user-images.githubusercontent.com/10614984/229649672-c1ed0110-99fb-4c6d-813f-e1fa19b86b68.png)
